doc/api/events.md

```diff
@@ -155,9 +155,60 @@ myEmitter.emit('error', new Error('whoops!'));
 // Prints: whoops! there was an error
 ```
 
+## AutoCatch of Promise rejection
+
+Using async functions in event handlers is problematic, because it would
+lead to `'unhandledRejection'` in case of an exception
+
+```js
+const ee = new EventEmitter();
+ee.on('something', async (value) => {
+  throw new Error('kaboom');
+});
+```
+
+The `autoCatch` option in the `EventEmitter` constructor or the global
+setting change this behavior, installing a `.catch()` handler on the
```

Comment:
i'm not exactly sure what to say here, but `catch()` isn't being called so there might need to be a change.